sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -367,6 +367,7 @@ class SessionCatalog(
       if (!ignoreIfExists) {
         throw new TableAlreadyExistsException(db = db, table = table)
       }
+      return
```

Comment:
Hm, this will disable external catalogs to handle on `ignoreIfExists=true`. Other external catalogs might have some logics on this call e.g.) sending a create table event.

Reply:
Oh, My bad, i will close the title.